Backend/app/history/views.py

```
@@ -1,3 +1,1072 @@
-from django.shortcuts import render
+from django.shortcuts import get_object_or_404
```

Ruff (0.12.2)

`django.shortcuts.get_object_or_404` imported but unused

Remove unused import: `django.shortcuts.get_object_or_404`

(F401)